src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/DSAKeyValue.cs

```diff
@@ -186,12 +186,12 @@ public override void LoadXml(XmlElement value)
                 Key.ImportParameters(new DSAParameters
                 {
                     P = (pNode != null) ? Convert.FromBase64String(pNode.InnerText) : null,
-                    Q = (pNode != null) ? Convert.FromBase64String(qNode.InnerText) : null,
+                    Q = (qNode != null) ? Convert.FromBase64String(qNode.InnerText) : null,
```

Comment:
I did wonder if that was the thinking behind it; I just thought it might be too implicit for someone who hasn't seen the code and spec to understand at first glance.